client/main.lua

```diff
@@ -126,6 +163,7 @@ function ResetPartial()
     })
 end
 
+---resets all injuries
```

Comment:
This is fairly pointless as the function name is already descriptive enough, possibly changing the function name to ```resetAllInjuries``` would be the best of both worlds on this matter. This is repeated throughout, but at the same time, not a necessary change.

Reply:
I agree. Renamed the functions for this and reset partial to be self documenting and removed the comments. I'll leave the rest alone for now, as I don't think it hurts, but they can be removed next time someone changes the function.